lib/fastlane/plugin/wpmreleasetoolkit/actions/common/set_milestone_frozen_marker_action.rb

```diff
@@ -52,7 +57,7 @@ def self.return_value
 
       def self.details
         # Optional:
```

Comment:
Nitpick: I guess the Fastlane template has this comment to let people know they don't have to add one, but given we added it, might as well remove it.

```suggestion
```

Or, since it's exactly the same as `description`, cash into it being optional and omitting it.

Reply:
Addressed in 80ea60473c1b52e23d25c4c1971626ec3298b1b4